lib/clients/teleport.go

```diff
@@ -98,7 +99,22 @@ func TeleportAuth(ctx context.Context, operator ops.Operator, proxyHost, cluster
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	return authClient, nil
+	return &AuthClient{
```

Comment:
I added this to reliably close the auth client since it was leaking connections and it prevented clean shutdown.